core/src/main/java/apoc/refactor/util/PropertiesManager.java

```diff
@@ -83,7 +83,9 @@ public static void combineProperties(Map.Entry<String, Object> prop, Entity targ
     private static Object createPropertyValueFromSet(Set<Object> input, RefactorConfig refactorConfig) {
         Object array;
         try {
-            if (input.size() == 1 && !refactorConfig.isSingleElementAsArray()) {
+            if (input.isEmpty()) {
+                return new int[0];
```

Comment:
If I remember correctly we create empty list as empty string arrays in Cypher (might be wrong). I don't think it makes a difference, certainly not in driver at least, maybe in core API. We could align here, but I'm also fine with keeping this, just wanted to point this out.